Add Layer Projector plugin

Shaion opened the pull request:
Layer-by-layer projection mode for projectors, similar to 3D printing.
     
     Features:
     - Display one layer at a time
     - Customizable axis (X, Y, Z)
     - Build restriction to active layer
     - Detailed statistics panel
     
     Repository: https://github.com/Shaion/LayerProjectorPlugin
     Author: Shaion

Pas2704 commented:
You should remove the package reference for Harmony. Package references in the manifest are for packages that aren't included with the launcher. The version you have is also different from the version Pulsar uses (2.3.3 vs 2.3.6).

Shaion commented:
Thanks for the feedback!

- Removed the Harmony NuGet reference from the plugin manifest.
- Updated the Harmony version in my project from 2.3.3 to 2.3.6 to match Pulsar's version.

Let me know if there's anything else that needs to be adjusted!

StarCpt commented:
Problems:
- Prev/Next layer buttons can de/increment the layer past the min/max layers
- Does not work with multigrid projector
- Terminal controls are recreated once I exit and re-load the world

StarCpt commented:
it also does not check if the current layer is within bounds when changing the layer axis or changing the blueprint

StarCpt commented:
Another potential bug: Changing the rotation (pitch/yaw/roll) of the projection changes the currently selected layer to the min value, unsure if this intended or not since changing the layer axis doesn't change it.
